docs/_includes/generated/security_configuration.html

```diff
@@ -32,6 +38,12 @@
             <td>Boolean</td>
             <td>Indicates whether to read from your Kerberos ticket cache.</td>
         </tr>
+        <tr>
+            <td><h5>security.module.factory.classes</h5></td>
+            <td style="word-wrap: break-word;">"org.apache.flink.runtime.security.modules.HadoopModuleFactory";<wbr>"org.apache.flink.runtime.security.modules.JaasModuleFactory";<wbr>"org.apache.flink.runtime.security.modules.ZookeeperModuleFactory"</td>
+            <td>List&lt;String&gt;</td>
+            <td>List of factories that should be used to instantiate security modules. All listed modules will be installed. Keep in mind that the configured security context might rely on some modules being present.</td>
```

Comment:
`configured security context might rely on some modules being present.` 
does this means we should throw an exception when installing module? or should we return false when `isCompatibleWith` is called?

Reply:
It's referring to the fact that for example the `HadoopSecurityContext` needs the `HadoopModule` to be present.